EKF/vel_pos_fusion.cpp

```diff
@@ -246,8 +246,6 @@ void Ekf::fuseVelPosHeight()
 		_state.quat_nominal.normalize();
 
 		// update covarinace matrix via Pnew = (I - KH)P
-		float KHP[_k_num_states][_k_num_states] = {};
```

Comment:
Agree, there is no need to declare a local variable for the Kalman gain and the class variable should be used. Note that changing this will have no effect on functionality as the Kalman gain does not need to be persistent.